lasagne/init.py

```diff
@@ -75,16 +170,51 @@ def sample(self, shape):
 
 
 class GlorotNormal(Glorot):
+    """Glorot with weights sampled from Uniform distribution
+
+    See Glorot for description of parameters.
+    """
     def __init__(self, gain=1.0, c01b=False):
         super(GlorotNormal, self).__init__(Normal, gain, c01b)
 
 
 class GlorotUniform(Glorot):
+    """Glorot with weights sampled from Uniform distribution
+
+    See Glorot for description of parameters.
+    """
     def __init__(self, gain=1.0, c01b=False):
         super(GlorotUniform, self).__init__(Uniform, gain, c01b)
 
 
 class He(Initializer):
+    """He weight initialization [1]_
+
+    Weights are initialized with std
+
+    :math:`\sigma = gain \sqrt{\frac{1}{fan_{in}}}`.
+
+    Parameters
+    ----------
+    initializer : Lasagne.init.Initializer
+        Initializer used to sample the weights. The default is Gaussian.
```

Comment:
Same suggestion as above to make clear you can't use just *any* initializer here, but only particular ones (they could be given a common base class, but we don't want to over-engineer stuff).